static/templates/settings/emoji_settings_admin.hbs

```diff
@@ -32,7 +32,7 @@
     <div class="progressive-table-wrapper" data-simplebar data-list-render="emoji_list">
         <table class="table table-condensed table-striped wrapped-table admin_emoji_table">
             <thead>
-                <th class="active" data-sort="alphabetic" data-sort-prop="name">{{t "Name" }}</th>
```

Comment:
I think we don't want to remove this -- we still want the user to be able to switch to sorting alphabetically if they want.

Reply:
Should we have a checkbox on the emoji settings page to ask them if they want theirs on top or mixed?

Reply:
Got it. It took me some time to get it. Resolved.